docker/s3/compose.yml

```diff
@@ -1,33 +1,27 @@
 version: '3.8'
 services:
   zookeeper:
-    image: kafka
-    build:
-      context: ../kafka/.
-      args:
-        - AK_REPO=https://github.com/aiven/kafka
-        - AK_BRANCH=3.3-2022-10-06-tiered-storage
-        - KAFKA_VERSION=3.3.2-SNAPSHOT
+    image: kafka-ts-s3-aws
     command:
       - zookeeper-server-start.sh
       - config/zookeeper.properties
 
   kafka:
-    image: kafka
+    image: kafka-ts-s3-aws
     build:
       context: ../kafka/.
       args:
         - AK_REPO=https://github.com/aiven/kafka
-        - AK_BRANCH=3.3-2022-10-06-tiered-storage
-        - KAFKA_VERSION=3.3.2-SNAPSHOT
+        - AK_BRANCH=3.0-2022-03-31-tiered-storage
```

Comment:
why getting back to 3.0 ?

Reply:
It's the fully working version atm, 3.3 has fetch broken https://github.com/aiven/kafka/issues/15

Reply:
let me make some notes.